install_config/redeploying_certificates.adoc

```diff
@@ -385,9 +385,17 @@ file:
 
 ----
 $ ansible-playbook -i <inventory_file> \
-    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-certificates.yml
+    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-master-certificates.yml
 ----
 
+[IMPORTANT]
+====
+After executing this playbook, you need to regenerate any xref:../dev_guide/secrets.adoc#service-serving-certificate-secrets[service signing certificate/key pairs] 
+by deleting existing secrets containing service serving certificates or removing and re-adding 
```

Comment:
containing should be that contain
Is there another way to say "service serving certificates?"

Reply:
@kalexand-rh It appears that  "service serving certificates" was the term when the feature was added to OpenShift. 
https://github.com/openshift/openshift-docs/pull/2324/files

Reply:
@mburke5678, that's fair. A bit confusing still, but fair. :)